doc/server/api_game/api.yaml

```diff
@@ -595,7 +595,21 @@ components:
           type: string
         location:
           $ref: '#/components/schemas/LocationDTO'
-      required: [ id, name, location ]
+        media_references:
+          type: array
+          items:
+            type: object
+            properties:
+              media_type:
+                type: string
+              title:
+                type: string
+              text:
+                type: string
+              media_reference:
+                type: string
+            required: [ media_type ]
```

Comment:
Would it make sense to put this into a separate object reference, as we have the exact same lines in Action and here or would this break things for you?

Reply:
I think that's a good idea. However, I've already done it here (but it hasn't been merged yet): https://github.com/MANVSim/MANVSim/commit/9d3d1ee4aa9eed00696ff704609b33ede9cdf395

 I would suggest that I rebase my changes onto your commits once yours has been merged, and then adapt it accordingly